browser/brave_origin/brave_origin_service_factory.cc

```diff
@@ -176,7 +176,10 @@ BraveOriginServiceFactory* BraveOriginServiceFactory::GetInstance() {
 BraveOriginServiceFactory::BraveOriginServiceFactory()
     : ProfileKeyedServiceFactory(
           "BraveOriginService",
-          ProfileSelections::BuildRedirectedInIncognito()) {}
+          ProfileSelections::Builder()
```

Comment:
Should we revert this change? It seems that[ `BuildRedirectedInIncognito` ](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/profiles/profile_selections.cc;l=97)already does this.